activerecord/lib/active_record/relation/finder_methods.rb

```diff
@@ -319,6 +319,21 @@ def exists?(conditions = :none)
       skip_query_cache_if_necessary { connection.select_rows(relation.arel, "#{name} Exists?").size == 1 }
     end
 
+    # Returns true if the relation contains the given record or false otherwise.
+    #
+    # No query is performed if the relation is loaded; the given record is
+    # compared to the records in memory. If the relation is unloaded, an
+    # efficient existence query is performed, as in #exists?.
+    def include?(record)
+      if loaded?
+        records.include?(record)
+      else
+        record.is_a?(klass) && exists?(record.id)
+      end
+    end
+
+    alias :member? :include?
```

Comment:
Worth extending test coverage to `#member?` as well?

Reply:
Done in 067e4e8.